### ISSUE TITLE
Storybook/Site: Implicit list of links

### ISSUE THREAD

**Issue by sirrah-tam** [ithaka/pharos#510]
**The problem**
The top level of the Sidenav component contains a set of visually grouped links that are not marked up as part of a list. Screen reader users will perceive the navigation region and then each individual link without giving context to how they are each associated. 

![Screen Shot 2023-04-07 at 10 53 40 AM|450](https://user-images.githubusercontent.com/6874453/230655204-09b43f38-0d32-4cac-a657-f9b21ea49163.png)
![Screen Shot 2023-04-07 at 10 55 56 AM|450](https://user-images.githubusercontent.com/6874453/230655197-8931126f-c474-4fc1-b4c1-4b1921e09af7.png)

**The solution**
Developers can add `<ul>` and `<li>` elements appropriately to give semantic structure to the top level of navigation.

**Additional information**
The menu items that are part of the collapsible sections do utilize this explicit list structure, the same practice should be used for all sets of grouped links.

**Comment by daneah** [comment]
We should fix the examples we have both on the Pharos site itself as well as in the story for this component, but the choice of the content is ultimately up to the consumer, so teams will also need to fix this themselves.
